backend/migrations/20240214154028_food_env_score.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Add up migration script here
+
+CREATE DOMAIN env_rating as smallint CHECK (VALUE > 0);
+CREATE DOMAIN env_value as smallint CHECK (VALUE > 0);
+
+CREATE TABLE food_env_score (
```

Comment:
what were your considerations for creating a new table here?
I think this is right/fine but I would like to hear some thoughts

Reply:
Creating a new table means not changuing the old ones

Reply:
Furthermore, this allows us to model the food:score relation as 1:[0,1] correctly